CRM/Upgrade/Incremental/php/FiveSeventyOne.php

```diff
@@ -44,4 +44,17 @@ public function upgrade_5_71_alpha1(string $rev): void {
     );
   }
 
+  /**
+   * Upgrade step; adds tasks including 'runSql'.
+   *
+   * @param string $rev
+   *   The version number matching this function name
+   */
+  public function upgrade_5_71_beta(string $rev): void {
+    $this->addTask('Make civicrm_event.selfcancelxfer_time', 'alterColumn', 'civicrm_event', 'selfcancelxfer_time',
+      "int(11) DEFAULT 0 NOT NULL COMMENT 'Number of hours prior to event start date to allow self-service cancellation or transfer.'"
+    );
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
```

Comment:
Wouldn't it need to run the sql first before changing the column to not null?

Reply:
@demeritcowboy yes! I just pushed up that change